Specs/Choice/Bulgarian/BooleanModel.json

```diff
@@ -0,0 +1,198 @@
+[
+  {
+    "Input": "Определено!",
+    "NotSupportedByDesign": "python",
```

Comment:
This PR only adds .NET support. 
Could you also add JavaScript? If not, please just add a NotSupported attribute for both JavaScript and Java.

Reply:
Do you mean to add NotSupported into Bulgarian\BooleanModel.json? (sorry if I haven't understood it right)

Reply:
Yes, basically add that attribute in each spec case in the json file. 
While Python is currently not supported by design and there is an attribute per test to show this  (there is no ChoiceRecognizer for Python yet), Javascript and Java have ChoiceRecognizer. 
So, until we have Bulgarian implementation, each test cases should be marked as NotSupported.

Reply:
Would you be able to contribute JavaScript implementation? If not, no worries. This is a separate question from the review.